aten/src/ATen/templates/TensorBody.h

```diff
@@ -1157,4 +1165,103 @@ inline c10::MaybeOwned<Tensor> Tensor::expect_contiguous(MemoryFormat memory_for
     return c10::MaybeOwned<Tensor>::owned(__dispatch_contiguous(memory_format));
   }
 }
+
+struct ViewMeta {
```

Comment:
Needs top level comment

Reply:
This and Alias are not referenced at all in TensorBody.h, so it would be better to place them in a separate header and include them only when they are needed. Will save you recompilation time.